src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -312,13 +322,15 @@ private void AssertValidConnection(IncomingConnectionInfo connectionInfo)
                     $"Cannot accept the incoming replication connection from {connectionInfo.SourceUrl}, because this node is in passive state.");
             }
 
-            if (_incoming.TryRemove(connectionInfo.SourceDatabaseId, out IncomingReplicationHandler value))
+            if (_incoming.TryGetValue(connectionInfo.SourceDatabaseId, out var value))
             {
+                if (value.LastHeartbeatTicks + TimeSpan.FromMinutes(1).Ticks > Database.Time.GetUtcNow().Ticks)
```

Comment:
Do we really need to calculate that with the usage of `Ticks`? :) 

- Can we use `new DateTime(value.LastHeartbeatTicks)` ?
- `TimeSpan.FromMinutes(1)` - please move to static readonly field

Reply:
done